src/qibocal/web/app.py

```diff
@@ -41,6 +41,11 @@ def get_graph(n, current_figure, url):
         url = f"/data{url}"
 
     method, folder, routine, qubit, format = url.split(os.sep)[2:]
+    try:
+        qubit = int(qubit)
+    except:
+        pass
```

Comment:
This qubit is passed to the plotting method. Right now I believe the expected types for the qubit are string or int, right? If yes, then maybe 
```suggestion
    if qubit.isdigit():
        qubit = int(qubit)
```
would also work and looks slightly better (avoids the global exception).

Reply:
I agree. Thanks for the suggestion!